src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -474,6 +475,7 @@ public void jnlpWorkingDir() throws Exception {
         r.assertBuildStatusSuccess(r.waitForCompletion(b));
     }
 
+    @Ignore("TODO fails in Kind with 'java.nio.file.AccessDeniedException: /home/jenkins/agent/workspace' from FilePath.mkdirs from FileMonitoringTask$FileMonitoringController.<init>")
```

Comment:
Caused by https://github.com/kubernetes-sigs/kind/issues/830. I wonder whether it is possible to use the one of the Azure provisioners on `ci.jenkins.io`

Reply:
We cannot use any cloud services from here, as they would require authentication.

I will check whether the Rancher provisioner works as in https://github.com/kubernetes-sigs/kind/issues/118#issuecomment-475134086.